src/App/App.js

```diff
@@ -8,17 +8,41 @@ class App extends Component {
     constructor() {
         super();
         this.state = {
-            movies: movieData.movies
+            movies: movieData.movies,
+            movie: null
         }
         console.log(this.state.movies)
     }
 
+    getMovieById = (id) => {
+        const filteredMovie = this.state.movies.find(movie => movie.id === id)
+        this.setState({movie: filteredMovie})
+        console.log("filteredMovies", filteredMovie)
+    }
+
+    switchState = () => {
+        this.setState({movie: null})
+    }
+
 render() {
+    if(this.state.movie) {
+        return (
+        <>
+        <Nav/>
+        <main className='movie-description'>
```

Comment:
This is a great start, we need to start to think about how to incorporate the other properties from the specific movie  API.